sw/airborne/modules/stereocam/state2camera/state2camera.c

```diff
@@ -32,8 +32,14 @@ static int frame_number_sending = 0;
 float lastKnownHeight = 0.0;
 int pleaseResetOdroid = 0;
 
+#ifndef STATE2CAMERA_SEND_DATA_TYPE
```

Comment:
Hi Felix,

I tried to look for pull request but Freek already accepted it I saw. I will add the comments on my next pull request, thanks for notifying me.


________________________________
From: Felix Ruess [notifications@github.com]
Sent: Sunday, May 15, 2016 1:23 PM
To: paparazzi/paparazzi
Cc: Kimberly Mcguire - LR; Author
Subject: Re: [paparazzi/paparazzi] Stereocam sendarray fix (#1662)


In sw/airborne/modules/stereocam/state2camera/state2camera.c<redir.aspx?REF=Fy6ijScVhbjFY16HZZJbBFI6srSst14NxNazg9CBxjMLLYJLuIrTCAFodHRwczovL2dpdGh1Yi5jb20vcGFwYXJhenppL3BhcGFyYXp6aS9wdWxsLzE2NjIjZGlzY3Vzc2lvbl9yNjMyOTAwOTE.>:

> @@ -32,8 +32,14 @@ static int frame_number_sending = 0;
>  float lastKnownHeight = 0.0;
>  int pleaseResetOdroid = 0;
>
> +#ifndef STATE2CAMERA_SEND_DATA_TYPE


A (doxygen) comment on what this define means would be nice...

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<redir.aspx?REF=-91MriXvTuHPvfb2nx2USrXxdEURMk6O1mKrRGyEr0ULLYJLuIrTCAFodHRwczovL2dpdGh1Yi5jb20vcGFwYXJhenppL3BhcGFyYXp6aS9wdWxsLzE2NjIvZmlsZXMvYTQwNDkyM2UyOTJlZTdiNmUzYTU0ODMyZDljMDViZThkNmRmZTE2YyNyNjMyOTAwOTE.>